resources/namespace-label-propagator-policy.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: policies.kubewarden.io/v1
+kind: ClusterAdmissionPolicy
+metadata:
+  name: psa-label-enforcer-policy
+spec:
+  policyServer: default
+  module: registry://ghcr.io/kubewarden/policies/psa-label-enforcer:v0.1.2
```

Comment:
Please use a policy  coming from the `test` namespace, this prevents the tests from messing with the download stats. If you need a specific policy to be under the `test` namespace I can push it

Reply:
Yes, I forget to do that. This is in my to do list but I forget it before pushing the changes. I'll try to do it. If I do not have the permissions, I let you know.

Reply:
https://github.com/orgs/kubewarden/packages/container/package/tests%2Fpsa-label-enforcer